app/Models/Image.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Image extends Model
+{
+    use SoftDeletes;
+    /**
+     * [$table description]
+     * @var string
+     */
+    protected $table= 'images';
+    /**
+     * [$dates description]
+     * @var array
+     */
+    protected $dates = ['deleted_at'];
+    /**
+     * [$fillable description]
+     * @var [type]
+     */
+    protected $fillable = [
+        'post_id',
+        'filename',
+        'created_at',
+        'updated_at'
+    ];
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Models/Image.php</br>Comma required after last value in array declaration